.gitignore

```diff
@@ -0,0 +1,22 @@
+# compiler output
+*.a
+*.o
+*.so
+*.so.*
+
+# compiled applications
+/progs/tcb_chkpwd
+/progs/tcb_convert
+/progs/tcb_unconvert
+
+# backup-files
+*~
+
+# archives
+*.tar
+*.tar.*
+
+# patch
+*.patch
+*.orig
+*.rej
```

Comment:
.gitignore should be in sync with "make clean".
See also #3.

Reply:
I'll drop my related changes in favor of #3.